src/dotnet/commands/dotnet-pack/NuGet/ManifestVersionUtility.cs

```diff
@@ -15,6 +15,7 @@ internal static class ManifestVersionUtility
         public const int TargetFrameworkSupportForDependencyContentsAndToolsVersion = 4;
         public const int TargetFrameworkSupportForReferencesVersion = 5;
         public const int XdtTransformationVersion = 6;
+        public const int ServiceableVersion = 8;
```

Comment:
This will throw right now since there are only 7 entries here. We still need to keep this equal to 7 , and add a comment that it needs to be changed to 8 when we integrate PackageType change here.